backend.go

```diff
@@ -132,6 +148,27 @@ func (b *kubeAuthBackend) role(ctx context.Context, s logical.Storage, name stri
 	return role, nil
 }
 
+func validateAliasNameSource(source string) error {
+	for _, s := range aliasNameSources {
+		if s == source {
+			return nil
+		}
+	}
+	return errInvalidAliasNameSource
+}
+
+func getAliasNameSourceDesc() string {
```

Comment:
Since the map only contains 2 entries and the number of options is unlikely expand to be more than these, I wonder if it makes things easier to read if we were to have a multi-line string literal for description instead of building the string this way. It's not [too uncommon](https://github.com/hashicorp/vault/blob/e14d2038735c43fb1264b6db7158ca8c49125819/builtin/credential/aws/path_role.go#L92) for descriptions to be lengthy and declared as such.

Reply:
Seems reasonable, updating now.

Reply:
Fixed in https://github.com/hashicorp/vault-plugin-auth-kubernetes/pull/110/commits/9eb89bd209fe72c4a4d05615eba02b1c9824f176